samples/native-image-sample/src/test/java/com/example/logging/ITNativeImageLoggingSample.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.logging;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ITNativeImageLoggingSample {
```

Comment:
This test is not in "Kokoro Samples" build.

https://source.cloud.google.com/results/invocations/26ead732-1ea4-4779-aa2e-1a1a6a09ab14/targets

Reply:
Yeah I was noticing this too. Hm not sure why it's not added to the Kokoro Samples build. Will have to look into this further.

Reply:
But the source-code comment of build.sh says the build runs if it's not associated with a pull request.

Reply:
You're right: https://github.com/googleapis/java-logging/blob/d2a125aa3bf91cfcd01e45d91fa116d645230dca/.kokoro/build.sh#L79. Looks like only ITs in snapshot/ are run in the presubmit build.